scenarios/openshift/node_scenarios_example.yml

```diff
@@ -1,18 +1,52 @@
 node_scenarios:
-  - actions:                                                        # node chaos scenarios to be injected
+  - actions:                                                        # Node chaos scenarios to be injected.
     - node_stop_start_scenario
     - stop_start_kubelet_scenario
     - node_crash_scenario
-    node_name:                                                      # node on which scenario has to be injected; can set multiple names separated by comma
-    label_selector: node-role.kubernetes.io/worker                  # when node_name is not specified, a node with matching label_selector is selected for node chaos scenario injection
-    instance_count: 1                                               # Number of nodes to perform action/select that match the label selector
-    runs: 1                                                         # number of times to inject each scenario under actions (will perform on same node each time)
-    timeout: 120                                                    # duration to wait for completion of node scenario injection
-    cloud_type: aws                                                 # cloud type on which Kubernetes/OpenShift runs
+    node_name:                                                      # Node on which scenario has to be injected.
+    label_selector: node-role.kubernetes.io/worker                  # When node_name is not specified, a node with matching label_selector is selected for node chaos scenario injection.
+    instance_count: 1                                               # Number of nodes to perform action/select that match the label selector.
+    runs: 1                                                         # Number of times to inject each scenario under actions (will perform on same node each time).
+    timeout: 120                                                    # Duration to wait for completion of node scenario injection.
+    cloud_type: aws                                                 # Cloud type on which Kubernetes/OpenShift runs.
   - actions:
     - node_reboot_scenario
     node_name:
     label_selector: node-role.kubernetes.io/infra
     instance_count: 1
     timeout: 120
-    cloud_type: aws
+    cloud_type: azure
+  - actions:
```

Comment:
this seems a little confsing to me to ahve both azure and baremetal in one file, should we separate into two separate files and then link to each from the node doc

Reply:
Done. Thanks.